src/test/java/com/google/devtools/build/lib/syntax/MethodLibraryTest.java

```diff
@@ -486,6 +486,7 @@ public void testRange() throws Exception {
         .testStatement("str(range(5, 0, -1))", "[5, 4, 3, 2, 1]")
         .testStatement("str(range(5, 0, -10))", "[5]")
         .testStatement("str(range(0, -3, -2))", "[0, -2]")
+        .testStatement("str(range(3)[1:2])", "[1]")
```

Comment:
Please add more tests combining range and slicing
https://github.com/google/skylark/blob/master/testdata/builtins.sky#L94